XIPs/xip-0-purpose-process.md

```diff
@@ -58,8 +66,8 @@ After vetting the idea as described above, your next responsibility is to presen
 
 ### XIP Process
 
-- **Idea** - An idea that is pre-draft. This is not tracked within the XIP Repository.
-- **Draft** - The first formally tracked stage of an XIP in development. An XIP is merged by an XIP Editor into the XIP repository when properly formatted.
+- **Idea** - An idea that is pre-draft. This is not tracked within the XIP Repository. To learn more, see [Get started with an idea](#get-started-with-an-idea).
+- **Draft** - The first formally tracked stage of an XIP in development. The idea author (now the XIP Author) creates the XIP Draft based on the guidelines and template discussed in the following sections. The XIP Draft is merged by an XIP Editor into the XIP repository when properly formatted.
```

Comment:
FYI - Draft to Review - will be the next phase we refine, as we did with Idea to Draft.